tools/accuracy_checker/accuracy_checker/metrics/semantic_segmentation.py

```diff
@@ -216,7 +213,10 @@ def update(self, annotation, prediction):
         result = np.zeros(shape=self.classes)
 
         annotation_data = annotation.mask
-        prediction_data = np.argmax(prediction.mask, axis=0) if self.use_argmax else prediction.mask.astype('int64')
```

Comment:
I think use_argmax should be stayed here, because not all models use resample postprocessing

Reply:
In that case you should use argmax in adapter